src/x11rb.rs

```diff
@@ -185,7 +188,7 @@ impl<C: HasConnection> X11rbServer<C> {
             )?
             .reply()?;
 
-        if reply.type_ != x11rb::NONE && (reply.type_ != AtomEnum::ATOM.into()) {
+        if reply.type_ != x11rb::NONE && (reply.type_ != Into::<u32>::into(AtomEnum::ATOM)) {
```

Comment:
Is this needed?

Reply:
It wasn't compiling with the latest `x11rb`... IIRC with the protocol split `x11rb` recently did a few things got moved around (probably by me, lmao) which probably caused that failure.

Reply:
But this PR doesn't bump up x11rb version how did you check the latest x11rb?

Reply:
Youre right, I run rust-analyzer with clippy and `-D warnings` on by default, and it flagged it for some reason. I'll revert that change tomorrow morning

Reply:
By the way, original code is not the best either. It might be good to rewrite as `u32::from(AtomEnum::ATOM)`.

Reply:
I changed it to `u32::from` in the latest commit.